temporian/implementation/numpy/operators/assign.py

```diff
@@ -18,23 +18,62 @@
 import numpy as np
 from temporian.implementation.numpy.data.event import NumpyEvent
 from temporian.implementation.numpy.data.event import NumpyFeature
+from temporian.implementation.numpy.data.sampling import NumpySampling
 
 
-def _get_equal_sampling_indexes(
-    sampling_1: "NumpySampling", sampling_2: "NumpySampling", index: tuple
+def _get_common_timestamps(
+    sampling_1: NumpySampling, sampling_2: NumpySampling, index: tuple
 ) -> np.ndarray:
-    """Get indexes of timestamps that are equal to sampling_1 & sampling_2
-      in a specific sampling index.
+    """It returns a np.ndarray with same shape as sampling_1.data[index] with True
+    values where sampling_1.data[index] == sampling_2.data[index], and False otherwise.
```

Comment:
This description doesn't seem correct - "It returns a np.ndarray with same shape as sampling_1.data[index] indicating if each value in it is present in sampling_2.data[index]" maybe? `==` would check positionally which isn't the case